nuntium/views.py

```diff
@@ -83,6 +83,18 @@ def dispatch(self, request, *args, **kwargs):
         self.writeitinstance = get_object_or_404(WriteItInstance, slug=request.subdomain)
         return super(WriteMessageView, self).dispatch(request=request, *args, **kwargs)
 
+    def get(self, *args, **kwargs):
+        # Check that the form contains valid data
```

Comment:
That's a good one! I'm thinking that perhaps we could separate this function into a different sort of ```is_valid``` thing that could perhaps do some other validations like for example rate limiting, but that could obviously go in another issue. I think that the idea would be to just hook validators into the form.